pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -127,9 +127,9 @@ type ReconcileHandler interface {
 type ReconcileNodeMaintenance struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
-	client      client.Client
-	scheme      *runtime.Scheme
-	drainer     *drain.Helper
+	client		client.Client
+	scheme		*runtime.Scheme
+	drainer		*drain.Helper
 	podInformer cache.SharedInformer
```

Comment:
This looks obviously wrong.
If you want to enforce tabs at the start of lines, that's worthwhile but it would be best if spaces within lines were left alone (go fmt already deals with that).

Reply:
target fix-tabs uses a tab stop of four, so every sequece of spaces that reach a column index of i%4 is replaced with tabs. Do you suggest a different value for tab stops?

Reply:
I suggest not touching anything other than whitespace prior to the first actual character.
In blocks such as this one, `go fmt` ensures that both the variable names and their types are lined up together.